docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -19,19 +19,19 @@ Meet the following software requirements before you use the team configuration f
    | v2.0.0-next.202107151328 | 7.0.0-next.202106012053 |
    | v2.0.0-next.tbd          | 7.0.0-next.202108202027 |
 
-2. [Initialize](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md#initializing-global-configuration) the Global Configuration file.
+2. Initialize the Global Configuration file by using either [the Zowe CLI](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md#initializing-global-configuration) or Zowe Explorer.
```

Comment:
@IgorCATech , do you think it makes sense to link to the `Create a Team Configuration File` section here? The Zowe CLI link will go to their repo but the Zowe Explorer link will go to the section below.